docs/types/resolvers.md

```diff
@@ -192,14 +192,15 @@ Info objects contain information for the current execution context:
 
 `class Info(Generic[ContextType, RootValueType])`
 
-| Parameter name  | Type                      | Description                                                           |
-| --------------- | ------------------------- | --------------------------------------------------------------------- |
-| field_name      | `str`                     | The name of the current field                                         |
-| context         | `ContextType`             | The value of the context                                              |
-| root_value      | `RootValueType`           | The value for the root type                                           |
-| variable_values | `Dict[str, Any]`          | The variables for this operation                                      |
-| operation       | `OperationDefinitionNode` | The ast for the current operation (public API might change in future) |
-| path            | `Path`                    | The path for the current field                                        |
+| Parameter name  | Type                                     | Description                                                           |
+| --------------- | ---------------------------------------- | --------------------------------------------------------------------- |
+| field_name      | `str`                                    | The name of the current field                                         |
+| context         | `ContextType`                            | The value of the context                                              |
+| root_value      | `RootValueType`                          | The value for the root type                                           |
+| variable_values | `Dict[str, Any]`                         | The variables for this operation                                      |
+| operation       | `OperationDefinitionNode`                | The ast for the current operation (public API might change in future) |
+| path            | `Path`                                   | The path for the current field                                        |
+| return_type     | `Optional[Union[Type, StrawberryUnion]]` | The return type of this operation                                     |
```

Comment:
is it optional?

Reply:
I copied the type from field_definition where it is optional, maybe it's wrong? 🤔

Reply:
actually, let's keep this hidden for now as we will change the type of `return_type` pretty soon